docs/operations/seed_settings.md

```diff
@@ -92,13 +92,25 @@ run for its operation.) Furthermore, load balancing may become imbalanced if mul
 
 Operators need to take these implications into account when considering switching external traffic policy to `Local`.
 
+### Proxy Protocol
+
+Traditionally, the client IP address can be used for security filtering measures, e.g. IP allow listing. However, for this to have any usefulness the client IP address needs to be correctly transferred to the filtering entity.
+
+Load balancers can either act transparently and simply pass the client IP on or they terminate one connection and forward data on a new connection. The latter (intransparant) approach requires a separate way to propagate the client IP address. Common approaches are an HTTP header for TLS terminating load balancers or [(HA) proxy protocol](https://www.haproxy.org/download/3.0/doc/proxy-protocol.txt).
+
+For level 3 load balancers, [(HA) proxy protocol](https://www.haproxy.org/download/3.0/doc/proxy-protocol.txt) is the default way to preserve client IP addresses. As it prepends a small proxy protocol header before the actual workload data, the receiving server needs to be aware of it and handle it properly. This means that activating proxy protocol needs to happen on both load balancer and receiving server at/around the same time as otherwise the receiving server will incorrectly interpret data as workload/proxy protocol header.
```

Comment:
```suggestion
For level 3 load balancers, [(HA) proxy protocol](https://www.haproxy.org/download/3.0/doc/proxy-protocol.txt) is the default way to preserve client IP addresses. As it prepends a small proxy protocol header before the actual workload data, the receiving server needs to be aware of it and handle it properly. This means that activating proxy protocol needs to happen on both load balancer and receiving server at/around the same time, as otherwise the receiving server will incorrectly interpret data as workload/proxy protocol header.
```

Reply:
Addressed with https://github.com/gardener/gardener/pull/9844/commits/1906b17ffb3af5dec86c603fabf6b7808030bcdd.